04_numeronym/anishaNair/main_test.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+var tests = []struct {
+	inputStrings  []string
+	outputStrings []string
```

Comment:
should probably be `expected` or `expectedOutput`

Reply:
done